apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/RODViewPoolCandidatePage.tsx

```diff
@@ -0,0 +1,523 @@
+    const snapshotCandidate = parsedSnapshot?.poolCandidates
+      ?.filter(notEmpty)
+      .find(({ id }) => id === poolCandidate.id);
+    const nonEmptyExperiences = parsedSnapshot.experiences?.filter(notEmpty);
+
+    mainContent = (
+      <>
+        <ApplicationInformation
+          pool={poolCandidate.pool}
+          snapshot={parsedSnapshot}
+          application={snapshotCandidate}
+        />
+        <div data-h2-margin="base(x2 0)">
+          <Accordion.Root type="single" mode="card" collapsible>
+            <Accordion.Item value="otherRecruitments">
+              <Accordion.Trigger>
+                {intl.formatMessage({
+                  defaultMessage: "Other recruitments",
+                  id: "kZs3Nk",
+                  description:
+                    "Heading for table of a users other applications and recruitments",
+                })}
+              </Accordion.Trigger>
+              <Accordion.Content>
+                <PoolStatusTable user={poolCandidate.user} pools={pools} />
+              </Accordion.Content>
+            </Accordion.Item>
+          </Accordion.Root>
+        </div>
+        <CareerTimelineSection experiences={nonEmptyExperiences ?? []} />
+      </>
+    );
+  } else if (snapshotUserPropertyExists && !preferRichView) {
+    mainContent = (
+      <>
+        {subTitle}
+        <pre
+          data-h2-background-color="base(background.dark.3)"
+          data-h2-border="base(1px solid background.darker)"
+          data-h2-overflow="base(scroll auto)"
+          data-h2-padding="base(x1)"
+          data-h2-radius="base(s)"
+        >
+          {JSON.stringify(parsedSnapshot, null, 2)}
+        </pre>
+      </>
+    );
+  } else {
+    mainContent = (
+      <NotFound headingMessage={intl.formatMessage(commonMessages.notFound)}>
+        <p>
+          {intl.formatMessage({
+            defaultMessage: "Profile snapshot not found.",
+            id: "JH2+tK",
+            description: "Message displayed for profile snapshot not found.",
+          })}
+        </p>
+      </NotFound>
+    );
+  }
+
+  return (
+    <>
+      <PageHeader
+        icon={UserCircleIcon}
+        subtitle={`${poolCandidate.user.firstName} ${
+          poolCandidate.user.lastName
+        } / ${getFullPoolTitleLabel(intl, poolCandidate.pool)}`}
+        navItems={pages}
+      >
+        {intl.formatMessage({
+          defaultMessage: "Candidate information",
+          id: "69/cNW",
+          description:
+            "Heading displayed above the pool candidate application page.",
+        })}
+      </PageHeader>
+      <ProfileDetails user={poolCandidate.user} />
+      <Sidebar.Wrapper>
+        <Sidebar.Sidebar>
+          <Heading>
+            {intl.formatMessage({
+              defaultMessage: "More actions",
+              id: "QaMkP7",
+              description:
+                "Title for more actions sidebar on view pool candidate page",
+            })}
+          </Heading>
```

Comment:
This should have `size="h3"` on it until we can resolve the heading sizes.

Reply:
Fixed by [a15ff54](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8853/commits/a15ff54d9bf756cdb1c3f86d344b1499222b2456)